SEC/Equities/EquityInstruments.rdf

```diff
@@ -577,6 +613,19 @@
 		<skos:definition>indicates a pre-arranged dividend rate</skos:definition>
 	</owl:ObjectProperty>
 	
+	<owl:ObjectProperty rdf:about="&fibo-sec-eq-eq;hasExDividendDate">
```

Comment:
These properties were in the original ontology that I collapsed, which is why I brought them over rather than deleting them altogether.  I've linked the two issues in JIRA and will close SEC-117 to this one.